lldb/source/Plugins/Language/Swift/SwiftFormatters.cpp

```diff
@@ -818,6 +820,16 @@ class TaskSyntheticFrontEnd : public SyntheticChildrenFrontEnd {
                    "could not get info for async task {0:x}: {1}", task_ptr,
                    fmt_consume(std::move(err)));
         } else {
+
+          // Print a backtrace of the Task to stdout.
+          ExecutionContext exe_ctx{m_backend.GetExecutionContextRef()};
+          auto tt = std::make_shared<ThreadTask>(
+              3000, task_info->resumeAsyncContext, exe_ctx);
+          StreamString ss;
+          tt->GetStatus(ss, 0, 100, 0, false, true);
```

Comment:
Is this where the unwinder gets invoked?

I was going to ask "why do we need the Task class to inherit from `Thread`, but the question above may be the answer to it.

Reply:
Correct, this is where the unwinder gets invoked. And yes, that's what the `ThreadTask` class exists.